sim/src/analytics.rs

```diff
@@ -192,24 +193,17 @@ impl Analytics {
 
         // Intersection delay
         if let Event::IntersectionDelayMeasured(trip_id, turn_id, agent, delay) = ev {
-            match agent {
-                AgentID::Car(_) => {
-                    if delay > Duration::seconds(30.0) {
-                        self.trip_intersection_delays
-                            .entry(trip_id)
-                            .or_insert_with(BTreeMap::new)
-                            .insert(turn_id, delay.inner_seconds() as u8);
-                    }
-                }
-                AgentID::Pedestrian(_) => {
-                    if delay > Duration::seconds(15.0) {
-                        self.trip_intersection_delays
-                            .entry(trip_id)
-                            .or_insert_with(BTreeMap::new)
-                            .insert(turn_id, delay.inner_seconds() as u8);
-                    }
-                }
-                AgentID::BusPassenger(_, _) => {}
+            let threshold = match agent {
+                AgentID::Car(_) => Duration::seconds(30.0),
+                AgentID::Pedestrian(_) => Duration::seconds(15.0),
+                // Don't record for riders
+                AgentID::BusPassenger(_, _) => Duration::hours(24),
```

Comment:
I missed the comment at first and 😱

Reply:
Buses are basically little mobile apartments, it's fine to chill in one for a day or so, right?